lib/generators/templates/prerequisites/node-version.tt

```diff
@@ -0,0 +1 @@
+<%= Suspenders::NODE_LTS_VERSION %>
```

Comment:
We need to [borrow from Rails here](https://github.com/rails/rails/blob/d65fec4fd2a047533408cbbd4824b248adc0e3fd/railties/lib/rails/generators/app_base.rb#L521-L529). Otherwise, we get the following error:

```
No preset version installed for command yarn
Please install a version by running one of the following:

asdf install nodejs 20.11.1
```

```suggestion
ENV.fetch("NODE_VERSION") do
  <%= `node --version`[/\d+\.\d+\.\d+/] %>
rescue
  <%= uspenders::NODE_LTS_VERSION %>
end
```

Reply:
#1201